server/build/babel/preset.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
           'next/head': require.resolve('../../../lib/head'),
           'next/document': require.resolve('../../../server/document'),
           'next/router': require.resolve('../../../lib/router'),
-          'styled-jsx/style': require.resolve('styled-jsx/style')
+          'styled-jsx/style': require.resolve('styled-jsx/style'),
+          'ansi-html': require.resolve('ansi-html')
```

Comment:
@nkzawa I tried adding module-resolver plugin to the nextPagesDir's babel-loader. But it seems like we can use only one module-resolver instance at a once. 

So, it'll discard all of these if I add a module-resolver there. That's why I added `ansi-html` to this list.